src/AureliaDependenciesPlugin.ts

```diff
@@ -55,8 +62,8 @@ class ParserPlugin {
     //    PLATFORM.moduleName("id");
     hooks.evaluate.tap("MemberExpression", TAP_NAME, expr => {
       if (expr.property.name === "moduleName" &&
-         (expr.object.type === "MemberExpression" && expr.object.property.name === "PLATFORM" ||
-          expr.object.type === "Identifier" && expr.object.name === "PLATFORM")) {
+         (String(expr.object.type) === "MemberExpression" && expr.object.property.name === "PLATFORM" ||
```

Comment:
Same question here and on next line: what can `expr.object.type` be that is not a string and would convert to `"MemberExpression"`?